modules/AlphaWalletFoundation/AlphaWalletFoundation/Analytics/AnalyticsTypes.swift

```diff
@@ -213,7 +213,8 @@ public enum Analytics {
         case dappBrowser
         case deepLink
         case tokenScript
-        case walletConnect
+        case walletConnectV1
```

Comment:
I think we should keep it as `walletConnect` but if you want to differentiate them, add a "type" property?

Rationale: so we keep the already-logged analytics compatible and also v1 is going away anyway, there wouldn't be much of v1 and would trend towards 0 soon.